pd-client/client.go

```diff
@@ -175,7 +175,7 @@ func (c *client) initClusterID() error {
 		time.Sleep(time.Second)
 	}
 
-	return errors.WithStack(errFailInitClusterID)
+	return errors.WithStack(errFailInitClusterID) // wrap error variable.
```

Comment:
oh, seem we don't need to add comments every where.

Reply:
In fact, the comments are added during the replacement. I was not sure if they should be included in the code base. The downside is it increases the cost of writing code. The advantage is that you can check if there is a reason for each `WithStack` with a simple grep command.

Reply:
Generally I prefer to have a system to understand this. In this PR we have comments. It is also possible to create wrapper functions for every external function call.

Reply:
These comments have too many types. It will make things complicated.